packages/web/src/components/gcds-details/gcds-details.css

```diff
@@ -1,13 +1,29 @@
-:host .gcds-details {
-  font: var(--gcds-details-font);
-
-  summary {
+:host {
+  .details__summary {
     position: relative;
     cursor: pointer;
-    display: inline-block;
+    display: block;
     padding: var(--gcds-details-summary-padding);
     color: var(--gcds-details-default-text);
     transition: background-color 0.15s ease-in-out, color 0.15s ease-in-out;
+    border-color: transparent;
+    background-color: transparent;
+    font: var(--gcds-details-font);
+    text-decoration: underline;
+    text-underline-offset: 0.2em;
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-details-default-decoration-thickness);
```

Comment:
You can remove this line and add it to the `text-decoration` on `line 12` instead:
`text-decoration: underline var(--gcds-details-default-decoration-thickness);`